internal/action/command.go

```diff
@@ -634,14 +634,19 @@ func (h *BufPane) ShowCmd(args []string) {
 	InfoBar.Message(option)
 }
 
+// Convert possible received string of raw escape sequence to a hexadecimal byte
+func convertRawEsc2Hex(rawEsc string) string {
```

Comment:
Just noticed this `2Hex` and it seems doesn't make much sense to me. What is a "hexadecimal byte"? It is just a byte.

...So now I have another pedantic suggestion: rename this function to something simple and "generic" like `parseKeyArg()` (i.e. without mentioning raw escape sequence in the function name) and move the comment into the function. Rationale: in most cases we do nothing to this string, we only convert it if it's a raw esc sequence, which is only a special corner case, so let's not emphasize this corner case too much in the code. Also it will make this function "extensible".

And the comment itself I would reword a bit to be more clear, something like "If this is a raw escape sequence, convert it to its raw byte form".

Reply:
Yep, more generic, more extensible without changing the function name again. :+1:
Done.